### PR TITLE
Feature/book creation

### PR DESCRIPTION
**Pull request by Raj-kadevar**

### CODE REVIEW

File: book/forms.py [hunk 1]
```diff
@@ -9,13 +9,14 @@ class Meta:
...
         price = self.cleaned_data['price']
         if price > 10000 or price <= 10:
             raise forms.ValidationError("price must be between 11 to 10,000")
         return price
+
+    # def clean_creator(self):
+    #     creator = self.cleaned_data['creator']
+    #     if creator is not :
+    #         raise forms.ValidationError("creator not matched")
+    #     return  creator
```
Comment (Dhruvi-Makwana) [inline book/forms.py]: remove commented code

File: book/urls.py [hunk 2]
```diff
@@ -5,8 +5,8 @@
     path('',views.LoginView.as_view(), name="login"),
     path('logout/',views.on_logout,name="logout"),
```
Comment (HARSHKALAL) [inline book/urls.py:7]: avoid write views everytime.

.views import <API name>.

File: book/urls.py [hunk 3]
```diff
@@ -5,8 +5,8 @@
...
     path('index/',views.index,name="index"),
-    path('delete/<int:id>',views.index,name="delete"),
-    path('update/<int:id>',views.index,name="update"),
+    path("delete/<int:id>/",views.delete,name="delete"),
+    path("update/<int:id>/",views.update,name="update"),
```
Comment (HARSHKALAL) [inline book/urls.py]: what update ? writer propoer name

**Review by HARSHKALAL — CHANGES_REQUESTED** [review]

File: book/templates/book/index.html [hunk 4]
```diff
@@ -27,8 +27,8 @@
             <td>{{book.author}}</td>
 
             {% if book.creator == request.user or book.author == request.user%}
-            <td> <a href="{% url 'delete' book.id %}"><button>Delete</button></a></td>
-            <td> <a href="{% url 'update' book.id %}"><button>Update</button></a></td>
+            <td> <a href="{% url 'deletebook' book.id %}"><button>Delete</button></a></td>
+            <td> <a href="{% url 'updatebook' book.id %}"><button>Update</button></a></td>
```
Comment (Dhruvi-Makwana) [inline book/templates/book/index.html:32]: form next time try to write delete-book or delete_book

File: book/views.py [hunk 5]
```diff
@@ -54,24 +54,29 @@ def post(self, request, *args, **kwargs):
             errors = book.errors
```
Comment (Dhruvi-Makwana) [inline book/views.py:49]: if you use LoginRequiredMixin then you don't have to write if condition

File: book/forms.py [hunk 6]
```diff
@@ -17,8 +15,6 @@ def clean_price(self):
 
 
 class UpdateFrom(forms.ModelForm):
```
Comment (Dhruvi-Makwana) [inline book/forms.py]: whose updateForm ? write BookUpdateForm instend

**Review by yatishTT — APPROVED** [review]

**Pull request merged**